### PR TITLE
Update falco-0-26-2.md

### PR DESCRIPTION
**Pull request by qinzhiben11**
Modify incorrect vocabulary

Signed-off-by: qinzhiben@uniontech.com

### CODE REVIEW

**Comment by qinzhiben11** [comment]
@mstemm @vjjmiras 
Please help to review. 
Thank you

File: content/zh-cn/blog/falco-0-26-2.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ spec:
 ```
 
 
-##接下来是什么?
+##接下来做什么?
```
Comment (vjjmiras) [inline content/zh-cn/blog/falco-0-26-2.md:77]:
```suggestion
## 接下来做什么?
```

**Review by vjjmiras — CHANGES_REQUESTED** [review]
Hey @qinzhiben11,

Thank you for your submission. I've realized there are many Markdown headers that were not correctly defined in this translation.
Like I suggested in your submission - it still requires you to approve it-, could you fix those too? Thank you in advance!

**Comment by qinzhiben11** [comment]
hi @vjjmiras ，Thank you for your suggestion. I have taken and made the necessary modifications.

**Comment by vjjmiras** [comment]
Thank you @qinzhiben11. Could you rebase your commits, so it can be merged?

**Review by vjjmiras — APPROVED** [review]
LGTM

**Pull request merged**